catalog/spatial/registry/registry-schema-bindings/src/test/java/org/codice/ddf/registry/schemabindings/RegistryPackageUtilsTest.java

```diff
@@ -0,0 +1,617 @@
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.registry.schemabindings;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasKey;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import javax.xml.bind.JAXBElement;
+
+import org.codice.ddf.parser.Parser;
+import org.codice.ddf.parser.ParserConfigurator;
+import org.codice.ddf.parser.ParserException;
+import org.codice.ddf.parser.xml.XmlParser;
+import org.junit.Before;
+import org.junit.Test;
+
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.AssociationType1;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.EmailAddressType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.ExtrinsicObjectType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.InternationalStringType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.OrganizationType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.PersonNameType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.PersonType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.PostalAddressType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryObjectListType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryObjectType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryPackageType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.ServiceBindingType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.SlotType1;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.TelephoneNumberType;
+
+public class RegistryPackageUtilsTest {
+    private Parser parser;
+
+    private ParserConfigurator configurator;
+
+    @Before
+    public void setUp() {
+        parser = new XmlParser();
+
+        configurator = parser.configureParser(Arrays.asList(RegistryObjectType.class.getPackage()
+                        .getName(),
+                RegistryPackageUtils.OGC_FACTORY.getClass()
+                        .getPackage()
+                        .getName(),
+                RegistryPackageUtils.GML_FACTORY.getClass()
+                        .getPackage()
+                        .getName()),
+                this.getClass()
+                        .getClassLoader());
+    }
+
+    @Test
+    public void testGetBindingTypesFromPackage() throws Exception {
+        RegistryObjectType registryObjectType = getRegistryObjectFromResource(
```

Comment:
Created the class variable.
There was no real reason I didn't go with that approach. Initially I was going to create a separate ebrim xml for testing specific cases but ended up just using the same one everywhere.